packages/fluentui/react-northstar/src/components/Tree/TreeTitle.tsx

```diff
@@ -101,21 +142,51 @@ const TreeTitle: React.FC<WithAsProp<TreeTitleProps>> & FluentComponentStaticPro
 
   const ElementType = getElementType(props);
   const unhandledProps = useUnhandledProps(TreeTitle.handledProps, props);
-
   const handleClick = e => {
     _.invoke(props, 'onClick', e, props);
   };
 
+  const selectIndicator = Box.create(
+    customSelectIndicator
+      ? {
+          selected,
+          selectable: selectable && !hasSubtree,
+          selectableParent,
+          expanded,
+        }
```

Comment:
Why are we sending this props to the Box shorthand? As far as I can see this props will just be propagated to the Box created. Am I missing something?